README.md

```diff
@@ -31,7 +31,7 @@ a decision about whether to accept or decline the proposal.
    should be addressed in a more detailed design.
 
 3. The proposal author writes a design doc to work out details of the proposed
-   design and address the concerns raised in the initial discussion.
+   design and addresses the concerns raised in the initial discussion.
```

Comment:
The original grammar here is correct.

It is (as originally): "The author writes a doc to work out and [to] **address**"
Not (as edited): "The author writes a doc and **addresses**"